integration-tests/integration_test.go

```diff
@@ -40,7 +40,8 @@ func TestRepeatedNetworkFlow(t *testing.T) {
 		NumIter:                11,
 		SleepBetweenCurlTime:   2,
 		SleepBetweenIterations: 1,
-		ExpectedReports:        []bool{true, false},
```

Comment:
I think it is good to check the order. When first working on these tests, I did have some more complicated tests for this. Perhaps in these three the order is not as important.